AGENTS.md

```
@@ -15,72 +15,26 @@ This is a Home Assistant custom integration for controlling smart adjustable bed
 
 ## Architecture
 
+Key modules (not an exhaustive listing — check the folder for the rest):
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)